awx/main/tasks/jobs.py

```
@@ -219,59 +220,51 @@ def __init__(self):
                     jwt = retrieve_workload_identity_jwt(
-                        self.instance,
+                        instance,
                         audience=input_src.source_credential.get_input('jwt_aud'),
                         scope=AutomationControllerJobScope.name,
                         workload_ttl_seconds=workload_ttl,
                     )
                     # Store token keyed by input source PK, since a credential can have
                     # multiple input sources (one per field), each potentially with a different audience
-                    credential_ctx[input_src.pk] = {"workload_identity_token": jwt}
+                    prep.workload_tokens[input_src.pk] = {"workload_identity_token": jwt}
                 except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)